assets/css/profile.scss

```diff
@@ -27,4 +33,53 @@ $tabs-link-hover-color: $red;
 .hero-background.is-transparent {
     opacity: 0.3;
 }
-  
\ No newline at end of file
+
+.skill-progress {
+    position: relative;
+}
+
+.progress-value {
+    position: absolute;
+    right: 0;
+    top: -100%;
+    transform: translateY(-15px);
+}
+
+.footer-wrapper {
+    text-align: center;
+}
+
+.biography {
+    z-index: 1;
+}
+
+.btn-group {
```

Comment:
I think you don't need this

https://bulma.io/documentation/components/dropdown/